cypress/page-objects/language-settings-page.js

```diff
@@ -0,0 +1,13 @@
+export class LanguageSettingsPage{
+    popupTextElement(){
+        return cy.get('.preview-text')
+    }
+
+    popupTitleElement(){
+        return cy.get('.language-message > .header')
+    }
+
+    headerElement(){
+        return cy.get('.header')
+    }
+}
```

Comment:
Missing a line break at the end of this file.

Reply:
fixed